Car_Localizer/Car_Localizer/Training/model_vehicle_classifier.py

```diff
@@ -0,0 +1,30 @@
+import os
+os.environ['KMP_DUPLICATE_LIB_OK'] = 'True'
+import torch.nn as nn
+
+
+#CNN Architecture:
+class VehicleClassifier(nn.Module):
+    def __init__(self):
+        super(VehicleClassifier, self).__init__()
```

Comment:
you do not need to repeat class name in super() method
super().__init__()

Reply:
Ok, it will be corrected in the next code release.